system_tests/test_vsphere_local_blueprints/local_linux_test.py

```diff
@@ -388,9 +761,37 @@ def cleanup_naming(self):
             task_retry_interval=3,
         )
 
+    def cleanup_naming_underscore(self):
```

Comment:
Embarrassingly it turns out that TestCase.addCleanup actually supports passing args & kwargs to the deferred function call, so `functools.partial` is completely unnecessary here.